src/DependencyInjection/Compiler/SetRouterPass.php

```diff
@@ -24,6 +24,7 @@ public function process(ContainerBuilder $container)
         // only replace the default router by overwriting the 'router' alias if config tells us to
         if ($container->hasParameter('cmf_routing.replace_symfony_router') && true === $container->getParameter('cmf_routing.replace_symfony_router')) {
             $container->setAlias('router', 'cmf_routing.router');
+            $container->getAlias('router')->setPublic(true);
```

Comment:
is setAlias not chainable with older symfony versions?

Reply:
@dbu No, it's [not even chainable in Symfony 3.3](https://github.com/symfony/symfony/blob/3.3/src/Symfony/Component/DependencyInjection/ContainerBuilder.php#L790).